tap/tests/test_tracker.py

```diff
@@ -221,6 +221,14 @@ def test_streaming_writes_plan(self):
 
         self.assertEqual(stream.getvalue(), '1..42\n')
 
+    @mock.patch('tap.tracker.ENABLE_VERSION_13', False)
+    def test_streaming_writes_plan_first(self):
```

Comment:
I think that setting the plan should also be made to work with the combined format.